### DISCUSSION TITLE
Code Search not working.

### DISCUSSION THREAD

**Discussion by RanarTheWrathful** [community/community#102061] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

"This repository's code has not been indexed yet. Try again later.
repo:RanarTheWrathful/the-wrathful-fixup [Whatever I search for]" I am getting this message when searching either of these repositories, any idea on how I can fix this?
Links:
https://github.com/RanarTheWrathful/the-wrathful-fixup
https://github.com/RanarTheWrathful/aps-plus-plus-assist

**Comment by ALICIALVO** [comment]
can you send a screenshot of the page when searching? maybe something in the fillter of your search is setup incorrectly
